src/models/Component.js

```diff
@@ -157,17 +225,27 @@ class Component extends FileInformation {
    * @param {string} [name] - Name of attribute to remove.
    */
   removeLinkAttribute(id, name = null) {
-    this.attributes = this.attributes
-      .filter((attribute) => {
-        if (name && attribute.name !== name) {
-          return true;
-        }
-        if (attribute.definition && attribute.definition.type === 'Link') {
-          return attribute.removeLink(id);
-        }
+    this.__removeLinkAttribute(this.attributes, id, name);
+  }
 
-        return true;
-      });
+  /**
+   * Remove all attributes refer to specify link.
```

Comment:
Remove all attributes related to the specified link ?

Reply:
done